prometheus-to-sd/main.go

```diff
@@ -160,6 +140,17 @@ func readAndPushDataToStackdriver(stackdriverService *v3.Service, gceConf *confi
 	exportTicker := time.Tick(*exportInterval)
 
 	for range time.Tick(*scrapeInterval) {
+		select {
```

Comment:
It's not very clear, why you have moved this code to begin of loop.

Reply:
Scraping errors could delay exporting, added a comment.